src/client/characters/OtherPlayer.ts

```diff
@@ -42,14 +33,12 @@ export default class OtherPlayer extends Player {
     }
   }
 
-  // preUpdate is called every frame for every game object
+  /* preUpdate is called every frame for every game object */
```

Comment:
If you are using jsdoc style comments, it begins with two asterisks:

```ts
/** preUpdate is called every frame for every game object. */
```

Reply:
Done!